src/tools/auth0/handlers/prompts.ts

```diff
@@ -1,11 +1,96 @@
-//@ts-nocheck because prompts haven't been fully implemented in this codebase yet
 import DefaultHandler from './default';
-import { Asset, Assets } from '../../../types';
+import { Assets, Language, languages } from '../../../types';
+import { isEmpty } from 'lodash';
 
-export const schema = { type: 'object' };
+const promptScreenTypes = [
+  'login',
+  'login-id',
+  'login-password',
+  'login-email-verification',
+  'signup',
+  'signup-id',
+  'signup-password',
+  'reset-password',
+  'consent',
+  'mfa-push',
+  'mfa-otp',
+  'mfa-voice',
+  'mfa-phone',
+  'mfa-webauthn',
+  'mfa-sms',
+  'mfa-email',
+  'mfa-recovery-code',
+  'mfa',
+  'status',
+  'device-flow',
+  'email-verification',
+  'email-otp-challenge',
+  'organizations',
+  'invitation',
+  'common',
+] as const;
+
+export const schema = {
+  type: 'object',
+  properties: {
+    universal_login_experience: {
+      type: 'string',
+      enum: ['new', 'classic'],
+    },
+    webauthn_platform_first_factor: {
+      type: 'boolean',
+    },
+    identifier_first: {
+      type: 'boolean',
+    },
+    customText: {
+      type: 'object',
+      properties: languages.reduce((acc, language) => {
+        return {
+          ...acc,
+          [language]: {
+            type: 'object',
+            properties: promptScreenTypes.reduce((acc, screenType) => {
+              return {
+                ...acc,
+                [screenType]: {
+                  type: 'object',
+                },
+              };
+            }, {}),
+          },
+        };
+      }, {}),
+    },
+  },
+};
```

Comment:
@sergiughf Similarly to the previous PR's comment, I made sure to validate this. It's especially important for prompts because the schema doesn't match the API responses exactly.